WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -606,6 +613,28 @@ def __rescheduleFailedJob( self, jobID, message, stop = True ):
     return self.__finish( 'Job Rescheduled', stop )
 
   #############################################################################
+  def __getCores( self ):
+    """
+    Return number of cores from gConfig and a boolean indicating corresponding to WholeNode option
+    """
+    tag = gConfig.getValue( '/Resources/Computing/CEDefaults/Tag', None )
```

Comment:
These infos are to be set for the JobAgent by the job pilot (which should be submitted with multicore configuration by the sitedirector).

I took the location @atsareg told me. Is there a more correct location?